### PR TITLE
Draft: Integrate firepilot

### PR DESCRIPTION
**Pull request by LeoFVO**
+ Integrate firepilot SDK to manage our micro VM
+ improve conditionnal execution about function and container

Closes (https://github.com/polyxia-org/project/issues/12)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+**This section is under construction.**
+
+## Run the project
+
+To run the project, the simplest is :
+
+    ```bash
+    make build
+    make run
+    ```
```
Comment (AlexandreBrg) [inline CONTRIBUTING.md:12]: This section should be in the readme

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+get_kernel_and_rootfs:
+	curl https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/x86_64/kernels/vmlinux.bin -o ./vmlinux.bin
+	curl https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/x86_64/rootfs/bionic.rootfs.ext4 -o ./rootfs.ext4
+
+build:
+	docker build -t test --build-arg ssh_prv_key="`cat ~/.ssh/id_rsa`" --build-arg ssh_pub_key="`cat ~/.ssh/id_rsa.pub`" -f ./riklet/Dev.Dockerfile .
```
Comment (AlexandreBrg) [inline Makefile:6]: I don't think it's a good idea to use the default SSH key, some setup have several SSH keys, and the default one doesn't match github credentials.

Moreover, this part might more suit the repository `project`.

File: MVP.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ curl https://s3.amazonaws.com/spec.ccfc.min/img/quickstart_guide/x86_64/rootfs/b
 
 cd rik
 
-git checkout firecracker-poc
+git checkout develop
```
Comment (AlexandreBrg) [inline MVP.md:17]: Same as my comment about repository `project` for this file, it doesn't belong to RIK for now.

**Review by AlexandreBrg — COMMENTED** [review]
First draw of a review here, cool job!

File: docker-compose.yml [hunk 4]
```diff
@@ -24,6 +24,11 @@ services:
   worker:
     container_name: rik-worker
     hostname: rik-worker
+    # TO BUILD THE IMAGE, YOU MUST RUN :
+    # docker build -t test --build-arg ssh_prv_key="$(cat ~/.ssh/id_rsa)" \
+    #   --build-arg ssh_pub_key="$(cat ~/.ssh/id_rsa.pub)" -f ./riklet/Dev.Dockerfile .
+    # because of private repo in cargo.toml
+    # Alternatively, you can use `make build` in the riklet folder
```
Comment (thomasgouveia) [inline docker-compose.yml:31]: Remove ?

**Review by thomasgouveia — CHANGES_REQUESTED** [review]
Remove all the SSH stuff please. As `firepilot` is fetched through the https URL of the repo, we don't need to specify any SSH key or whatever else.

For the rest, I'll wait for the demo but the code LGTM

**Pull request merged**